front_end/.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
```

Comment:
is this gonna work from this directory or do we need to modify the top level .travis.yml to do front end tests?